packages/docsearch-css/src/modal.css

```diff
@@ -1,5 +1,14 @@
 /* Body modifier */
 
+/* Prevent body scroll on modal-open for mobile */
+/* https://stackoverflow.com/a/24727206/6276948 */
+@media (max-width: 768px) {
+  body:has(.DocSearch-Container) {
+    overflow: hidden;
+    position: fixed;
+  }
```

Comment:
Shouldn't this be colocated with the other mobile styles, instead of repeating `@media (max-width: 768px)`?

Reply:
Fair. Fixed.